test/service/test_manifest.py

```diff
@@ -2038,3 +2043,45 @@ def test_compact_manifest(self):
             )
         ]
         self._assert_tsv(expected, response)
+
+    @unittest.skipIf(not config.enable_replicas,
+                     'The format is replica-based')
+    @manifest_test
+    def test_jsonl_manifest(self):
```

Comment:
Test method names are still inconsistent. The commit whose title claims that it renames methods, doesn't rename any.